c-dependencies/xqd.h

```diff
@@ -294,6 +298,18 @@ WASM_IMPORT("fastly_dictionary", "get")
 int xqd_dictionary_get(DictionaryHandle dict_handle, const char *key, size_t key_len, char *value,
                        size_t value_max_len, size_t *nwritten);
 
+// Module fastly_kv
+#ifdef ENABLE_OBJECT_STORE
+WASM_IMPORT("fastly_kv", "open")
+int fastly_kv_open(const char *name, size_t name_len, ObjectStoreHandle *object_store_handle_out);
+WASM_IMPORT("fastly_kv", "lookup")
+int fastly_kv_lookup(ObjectStoreHandle object_store_handle, const char *key, size_t key_len,
+                     BodyHandle *opt_body_handle_out);
+WASM_IMPORT("fastly_kv", "insert")
+int fastly_kv_insert(ObjectStoreHandle object_store_handle, const char *key, size_t key_len,
+                     BodyHandle body_handle, uint32_t max_age, size_t *inserted);
+#endif
```

Comment:
These are the now outdated hostcalls, once the new ones are released I can update this pull-request to use the new ones